### DISCUSSION TITLE
Welcome to capa-rules Discussions!

### DISCUSSION THREAD

**Discussion by williballenthin** [fireeye/capa-rules#222] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

(this is willi:) I've clicked through the default Discussions config to create this. Let's see what we can use this Github feature for.

**Comment by ggold7046** [comment]
Hi @williballenthin, I have found some files which is mentioned in https://github.com/mandiant/capa-rules/issues/220 .   `communication/icmp/send-icmp-echo-request.yml` , `nursery/resolve-function-by-fnv-1a-hash.yml`
Could you please guide me if these files need to be updated ? 
And if these files need to be updated, could you please guide me how to update this using `lib` rule ?

**Reply by williballenthin** [reply]
please use the existing issue thread for discussing the necessary changes

**Comment by Dextera0007** [comment]
Learning capa rules, objective is to analyze various packets using capa rules: PE, ELF and .net files any help you can provide would be greatly appreciated. Here are the errors below when attempting to run  a PE files  :

capa -s capa-master\sigs -r capa-rules-master file.exe
error:capta: Input file does not appear to be a PE file or ELF file 
error:capa: capa currently only supports analyzing PE and ELF files (or shellcode, when using --fomrat sc32|sc64

**Reply by mr-tz** [reply]
Please open an issue here: https://github.com/mandiant/capa/issues and share the file hash (if possible).

**Comment by Iroxious** [comment]
Thank you very much for your contributions.
May I ask if you have any plans in the near future to synchronize capa rules with ATT&CK v14.1?

**Reply by mr-tz** [reply]
Thank you, tracking here: https://github.com/mandiant/capa-rules/issues/887
